packages/libclc/build.sh

```diff
@@ -0,0 +1,85 @@
+TERMUX_PKG_HOMEPAGE=https://libclc.llvm.org/
+TERMUX_PKG_DESCRIPTION="Open source implementation of the library requirements of the OpenCL C programming language"
+TERMUX_PKG_LICENSE="Apache-2.0, NCSA"
+TERMUX_PKG_LICENSE_FILE="libclc/LICENSE.TXT"
+TERMUX_PKG_MAINTAINER="@termux"
+LLVM_MAJOR_VERSION=20
+TERMUX_PKG_VERSION=(
+	${LLVM_MAJOR_VERSION}.1.7
+	${LLVM_MAJOR_VERSION}.1.3
+)
+TERMUX_PKG_SRCURL=(
+	https://github.com/llvm/llvm-project/releases/download/llvmorg-${TERMUX_PKG_VERSION[0]}/llvm-project-${TERMUX_PKG_VERSION[0]}.src.tar.xz
+	https://github.com/KhronosGroup/SPIRV-LLVM-Translator/archive/refs/tags/v${TERMUX_PKG_VERSION[1]}.tar.gz
```

Comment:
@finagolfin I am splitting libclc to separate package not merging into libllvm unless you are ok with adding external dependency